src/App.tsx

```diff
@@ -125,15 +125,35 @@ function App() {
     }
   }, [onlyBombs, appState]);
 
-  const neighbors = board.cells.map((_, i) => countNeighbors(board, i));
+  const neighbors = board.cells.map((_, i) => {
+
+    const count = countNeighbors(board, i);
+
+    if (count > 0) {
+
+      let countIterator = count;
+      let iterators = 3;
+
+      while (countIterator > 3) {
+        countIterator = countIterator - 3;
+        iterators--;
+      }
+
+      const colorLevel = (255 / 3) * iterators;
+      const color = `rgb(${(+(countIterator === 1)) * colorLevel}, ${(+(countIterator === 2)) * colorLevel}, ${(+(countIterator === 3)) * colorLevel})`;
```

Comment:
Instead of trying to be clever with going back and forth between booleans and integers, can we just have a function that returns known colors? It doesn't have to be calculated.

Reply:
I'm okay with it not being 100% accurate. Please do not calculate it.